CefSharp.Test/DevTools/DevToolsClientFacts.cs

```diff
@@ -156,5 +157,90 @@ public async Task CanUseMultipleDevToolsClientInstancesPerBrowser()
                 }
             }
         }
+
+        [Fact]
+        public async Task CanTakeMultipleScreenshotAsync()
+        {
+            Form snapForm = new Form();
+            snapForm.Size = new System.Drawing.Size(800, 600);
+            Panel p = new Panel();
+            p.AutoScroll = true;
+            p.Dock = DockStyle.Fill;
+
+            using (var browser = new CefSharp.WinForms.ChromiumWebBrowser("www.google.com"))
+            {
+                p.Controls.Add(browser);
+                snapForm.Controls.Add(p);
+                snapForm.Show();
+
+
+                browser.LoadPageAsync();
+                browser.IsBrowserInitializedChanged += (sender, e) =>
+                {
+                    System.Diagnostics.Trace.WriteLine(browser.GetDevToolsClient().ToString());
+
+                    if (browser.IsBrowserInitialized)
+                    {
+                        output.WriteLine("Browser Initialized");
+                        Cef.UIThreadTaskFactory.StartNew(() =>
```

Comment:
Seems totally unrelated to testing DevTools, please remove.

Reply:
```requestContext``` code is removed